org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/impl/IToBeBuiltComputerContribution.java

```diff
@@ -59,4 +62,17 @@
 	 * should not be processed by the Xtext builder.
 	 */
 	boolean isRejected(IFolder folder);
+	
+	/**
+	 * Add all the projects to the result, that may affect the resources in this projects.
+	 * The XtextBuilder will return the collected set of projects as the interesting projects after
+	 * each build round.
+	 * 
+	 * @see IncrementalProjectBuilder#build
+	 * 
+	 * @since 2.19
+	 */
+	default void addInterestingProjects(IProject thisProject, Set<IProject> result) {
```

Comment:
am not sure if i like this name but i dont have a better yet

Reply:
It's the term used by `IncrementalProjectBuilder.getInterestingProjects`